shared/chat/conversation/list-area/normal/index.stories.js

```diff
@@ -0,0 +1,214 @@
+// @flow
+import React from 'react'
```

Comment:
renders some mock messages and can be turned into a more interactive mode. uses a prng so its static